### PR TITLE
Pack/stack operation

### PR DESCRIPTION
**Pull request by kbalka**

### CODE REVIEW

**Comment by crlishka** [comment]
Rerunning using updated CI:

**Comment by crlishka** [comment]
TESTNOW

File: src/ngraph_builder.cc [hunk 1]
```diff
@@ -1273,6 +1273,67 @@ tf::Status Builder::TranslateGraph(const std::vector<tf::TensorShape>& inputs,
...
+        return tf::errors::InvalidArgument(
+            "Number of inputs should be at least 1 for Pack");
+      }
+
+      ng::NodeVector ng_concat_inputs;
+
+      for (size_t i = 0;i < op->num_inputs();++i) {  
+        tf::Node* tf_input;
+        TF_RETURN_IF_ERROR(op->input_node(i, &tf_input));
+        auto ng_input = ng_op_map.at(tf_input->name());
```
Comment (avijit-nervana) [inline src/ngraph_builder.cc]: Need a try catch here. The lookup will throw exception - you've done this before - so you know what to do :)

**Comment by crlishka** [comment]
Rerunning using TESTNOW, to test new notification features for CI pre-merge testing:

**Comment by crlishka** [comment]
TESTNOW

**Comment by crlishka** [comment]
TESTNOW

**Comment by crlishka** [comment]
TESTNOW

**Pull request merged**